### ISSUE TITLE
IE9 and Image issue

### ISSUE THREAD

**Issue by simonmacfadyen** [kottenator/jquery-circle-progress#71]
Hey there,

I have a strange (possibly bug), in IE9. The implementation @ http://codepen.io/anon/pen/wMKNBB results in the canvas drawing successfully on load e.g. http://i.imgur.com/ZZqw36w.png

However, when I use the method for redraw using the below:

```javascript
// Demo use only - redraw circle on click for testing purposes
$( '.circle-wrapper button' ).on( "click", function() {
	// Redraw the circle using API function 'redraw'
	$(this).parent().find('.circle').circleProgress('redraw');
});
```
I receive the below visual bug (Image does not appear): http://i.imgur.com/FTYWYOb.png

**Comment by kottenator** [comment]
Thank you for the report. I'll check what's wrong

**Comment by simonmacfadyen** [comment]
Great, thanks :+1:

**Comment by simonmacfadyen** [comment]
So kottenator, do you think this is a problem in ie?

**Comment by kottenator** [comment]
I've tested this bug in IE11 (in emulated Document mode: IE9) and I can't reproduce it, it works.
The screenshot that you've attached - it may happen when image was not loaded ... I think I'll give it another try in virtual machine with native IE9 ...

**Comment by kottenator** [comment]
@simonmacfadyen - I have 2 news for you, and both are good ;)

1. I've reproduced the bug in IE9 on Windows 7 (VM) - it happens randomly after few "redraw"s (but not obligatory). I'm not 100% sure but most likely it's because `Image.onload` doesn't work stable in IE9 (currently background image is [re-loaded on each "redraw"](https://github.com/kottenator/jquery-circle-progress/blob/1.1.3/dist/circle-progress.js#L224))
2. I've created [a code snippet with fix for IE9](http://jsbin.com/zawugu/3/edit?html,js,output). The idea is to cache `Image` object after first load
